pythran/pythonic/include/types/numpy_broadcast.hpp

```diff
@@ -176,6 +179,17 @@ namespace pythonic
       {
         return {_base._value};
       }
+#ifdef USE_BOOST_SIMD
+      using simd_iterator = const_broadcast_iterator<decltype(_base._splated)>;
+      simd_iterator vbegin() const
+      {
+        return {_base._splated};
+      }
+      simd_iterator vend() const
+      {
+        return {_base._splated};
+      }
+#endif
```

Comment:
On Tue, May 10, 2016 at 12:21:28PM -0700, pbrunet wrote:
> > @@ -176,6 +179,17 @@ namespace pythonic
> >        {
> >          return {_base._value};
> >        }
> > +#ifdef USE_BOOST_SIMD
> > +      using simd_iterator = const_broadcast_iterator<decltype(_base._splated)>;
> > +      simd_iterator vbegin() const
> > +      {
> > +        return {_base._splated};
> > +      }
> > +      simd_iterator vend() const
> > +      {
> > +        return {_base._splated};
> > +      }
> > +#endif
> 
> I don't get it.
It's hacky : this iterator always returns the same value and cannot
advance, or be compared to the end...